snuba/datasets/dataset_schemas.py

```diff
@@ -25,14 +25,6 @@ def __init__(
     def get_read_schema(self) -> Schema:
         return self.__read_schema
 
-    def get_write_schema(self) -> Union[WritableTableSchema, None]:
```

Comment:
I would like this to become just a DDL repository, and maybe orchestrate the the production migrations.
For now I could remove the get_write_schema features so there is no assertion anymore.